go.mod

```diff
@@ -18,4 +14,26 @@ require (
 	gopkg.in/natefinch/lumberjack.v2 v2.0.0
 )
 
-go 1.13
+require (
+	github.com/golang/snappy v0.0.4 // indirect
+	github.com/jmespath/go-jmespath v0.4.0 // indirect
```

Comment:
What’s the reason for so many new lines here? Or is it automatic or smt?

Reply:
yes, they were automatically generated

Reply:
Calm